src/Converters/Test/YesNoConverterTest.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
-namespace PDGA\DataObjects\Converters;
+namespace PDGA\DataObjects\Converters\Test;
```

Comment:
Is this change (to move all the test classes into their own `Test` directory) part of the coding standards?

Reply:
Not directly. Some of the test class files had inner classes defined to help with testing. Those inner classes had to be moved to their own file and it didn't seem appropriate to have those test specific classes mixed in with everything else. So, moved those to `Test` namespaces and I also took a liberty here to make it all consistent.